QualityControl/lib/dtos/ObjectGetDto.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import Joi from 'joi';
+
+const periodNamePattern = /^LHC\d{1,2}[a-z]+$/i;
+
+/**
+ * Creates a set of Joi validation schemas for object-related DTOs (Data Transfer Objects)
+ * @param {string[]} runTypes - Array of valid run types to be used for validation
+ * @returns {object} An object containing multiple Joi validation schemas:
+ *   - ObjectsGetDto: Schema for getting multiple objects
+ *   - ObjectContentsGetDto: Schema for getting object contents
+ *   - ObjectGetByIdDto: Schema for getting an object by ID
+ *   - qcgIdDto: Schema for validating object ID in URL
+ */
+export function createObjectGetDtos(runTypes) {
```

Comment:
I would use an options object instead of a single parameter like runTypes because it makes the function more flexible and easier to extend in the future
```suggestion
export function createObjectGetDtos({runTypes}) {
```